components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -1545,6 +1545,8 @@ - (void)setErrorText:(NSString *)errorText
     // If there is a saved state, use it.
     if (self.previousLeadingText) {
       self.textInput.leadingUnderlineLabel.text = self.previousLeadingText;
+      UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification,
```

Comment:
I would have expected this to be at done on ln 1592.
It is under the heading of "accessibility".

also the accessible labels haven't been set yet. so posting a notification before the final values are set might produce the wrong values.